about/discord-roles.md

```diff
@@ -0,0 +1,32 @@
+
+One of the things that makes The Odin Project a unique place on the internet is the vibrant community on Discord. We help each, hang out together, and most importantly, build each other up. 
+
+Building communities is difficult and we want to make sure that we encourage the right kinds of behaviors. When you join our Discord server, you will notice a few roles. Not all of them are roles that are essential, but they are used to help organize events (such as book clubs). 
+
+## Essential Discord User Roles:
+
+- **Moderator** - individuals with this role are active community members that are helping moderate the Discord server. They enforce the rules, delete spam, and encourage the types of interactions we want within our community. 
+- **Maintainer** - individuals with this role were invited to take a more active role in supporting The Odin Project. These folks also moderate the Discord community, but additionally they provide feedback on Github issues and Pull Requests. Maintainers also support TOP by writing parts of the curriculum.
+- **Core** - individuals with this role are the leadership group behind The Odin Project. Core team members have volunteered thousands of hours to maintain and grow TOP. 
+
+It is important to specify that **there is no application process** for any of the roles above. Every single person that has one of the roles above has been invited due to who they are as an individual and the support they have provided out of their own desire to the growth of The Odin Project. 
+
+
+## Secondary Discord User Roles
+
+We all know that as a Sever Booster you receive a special role in the server. In TOP Discord server we also use roles to highlight achievements within TOP and  participation in events. 
+
+These roles have **no moderation** privileges.
+
+- **club-40** - folks that have been around TOP for a while and have helped out other students in the channels. This role is maintained by the Odin Bot and this status is achieved by reaching 40 points. 
+- **alumni** - members of this group have gone through TOP curriculum and landed a job! Check some of their stories in #success-stories channel. 
+- **cah-crew** and **jam participant** - community events focused on providing opportunities for team work. Keep an eye on the #announcements for more events!
+- **...-book-club** - participants in our book clubs, such as YDKJS book series. 
+
+
+## Discretionary Roles
+These roles have **no moderation privileges**. They are used to highlight specific background our community members have. 
+
+
+- **Professional** - We have members of our community that are here to help others to grow, because they are employed in the software engineering role. We wanted to highlight those individuals so that their experiences in the workplaces helps others make more educated decisions. This role is up to the **discretion by the Core team**. It does require a verification step, and the admittance is *not guaranteed*. 
```

Comment:
```suggestion
- **Professional** - We have members of our community that are here to help others to grow, because they are employed in a software engineering role. We wanted to highlight those individuals so that their experiences in the workplaces helps others make more educated decisions. This role is up to the **discretion of the Core team**. It does require a verification step, and the admittance is *not guaranteed*. 
```